src/commands/frameworks-backends-list.ts

```diff
@@ -27,11 +20,11 @@ export const command = new Command("backends:list")
       style: { head: ["green"] },
     });
     table.colWidths = COLUMN_LENGTH;
-    const backendsList: gcp.ListBackendsResponse[] = [];
+    const backendsList: gcp.Backend[] = [];
     try {
       const backendsPerRegion = await gcp.listBackends(projectId, location);
-      backendsList.push(backendsPerRegion);
-      populateTable(backendsPerRegion, location, table);
+      backendsList.push(...backendsPerRegion.backends);
```

Comment:
just ooc what do we do if the user has 100s of backends? It doesn't look like we allow them to page through the results atm

Reply:
good point. will leave a bug for myself.